docs/products/clickhouse/concepts/clickhouse-tiered-storage.rst

```diff
@@ -0,0 +1,83 @@
+
+The tiered storage in Aiven for ClickHouse consists of the following two layers:
+
+SSD - the first tier
+  Fast storage device with limited capacity, better suited for fresh and frequently queried data, relatively costly to use
+
+Object storage - the second tier
+  Affordable storage device with unlimited capability, better suited for historical and more rarely queried data, relatively slower
+
+Why use it
+----------
+
+By :doc:`enabling </docs/products/clickhouse/howto/enable-tiered-storage>` and properly :doc:`configuring </docs/products/clickhouse/howto/configure-tiered-storage>` the tiered storage in Aiven for ClickHouse, you can use storage resources efficiently and, therefore, significantly reduce costs of storing data ingested into an Aiven for ClickHouse instance.
+
+How it works
+------------
+
+With the tiered storage :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>`, by default Aiven for ClickHouse stores data on your SSD until it reaches 80% of its capacity. After exceeding this size-based threshold, the data is stored in the object storage.
+
+Optionally, you can :doc:`configure the time-based threshold </docs/products/clickhouse/howto/configure-tiered-storage>` in the tiered storage. Based on the time-based threshold, the data is moved from your SSD to the object storage after a specified time period.
+
+.. mermaid:: 
+
+    sequenceDiagram
+        Application->>+SSD: writing data
+        SSD->>Object storage: moving data based <br> on storage policies 
+        par Application to SSD
+            Application-->>SSD: querying data
+        and Application to Object storage
+            Application-->>Object storage: querying data
+        end
+        alt if stored in Object storage
+            Object storage->>Application: reading data
+        else if stored in SSD
+            SSD->>Application: reading data
+        end
+
+.. note:: 
+    
+    Backups are taken for data that resides both on SSD and in object storage.
+
+Typical use case
+----------------
+
+In your Aiven for ClickHouse service, there is a significant amount of data that is there for a while and is hardly ever accessed. It's stored on SSD and, thus, high-priced. You decide to :doc:`enable </docs/products/clickhouse/howto/enable-tiered-storage>` the tiered storage for your service to make your data storage more efficient and reduce the costs. For that purpose, you select a plan for your service that supports the tiered storage and you :doc:`enable </docs/products/clickhouse/howto/enable-tiered-storage>` the feature on particular tables. You :doc:`configure </docs/products/clickhouse/howto/configure-tiered-storage>` the time-based threshold for controlling how your data is stored in the two layers.
```

Comment:
Is 'the tiered storage' right here, or should it just be 'tiered storage'? The latter sounds better to me.

Reply:
My idea is that 
1. In `the tiered storage feature`, `tiered storage` is a proper noun, a name of the feature, so it's like `the pgvector extension`. In this context, `tiered storage` on its own (without `feature`) should not have an article.
2. But on top of `tiered storage` being a proper noun, depending on the context it can also be a common noun denoting a storage that is of a tiered nature. In this context, `tiered storage` should have some article, depending on the context (definite/indefinite).

Based on the assumptions above, I'm trying to use the articles depending on the context. And yes, it's challenging sometimes to differentiate if you mean the feature name here or just the storage itself.

Hope that makes sense? I'll try to revisit the content and adjust the use of articles as needed. Please let me know if you see any other inconsistencies.